src/Components/InstanceTypes/SuggestedInstanceTypesTable.js

```diff
@@ -48,39 +49,39 @@ export default function SuggestedInstanceTypesTable() {
                                 onPerPageSelect
                             }}
                         />
-                        {!loading ? <TableComposable
-                            aria-label='suggested instance types table'
-                            variant={TableVariant.compact}
-
-                        >
-                            <Thead noWrap>
-                                <Tr>
+                        { serverError ? <ErrorState/> :
```

Comment:
@r14chandra do you think we need this validation both the places or just in the suggestedInstance type page will be enough?

Reply:
Yes, either of the apis may get errors while calling.